documentation/configuration/azure-ad-authentication-configuration.md

```diff
@@ -0,0 +1,56 @@
+
+### Was this documentation helpful? [Share feedback](https://www.research.net/r/DGDQWXH?src=documentation%2Fconfiguration%2Fazure-ad-authentication-configuration)
+
+# Azure Active Directory Authentication Configuration (8.0+)
+
+Azure Active Directory authentication must be configured before `dotnet monitor` starts, it does not support being configured or changed at runtime.
+
+## Configuration Options
+
+| Name | Type | Required | Description |
+|---|---|---|---|
+| ClientId | string | true | The unique application (client) id assigned to the app registration in Azure Active Directory. |
+| RequiredRole | string | true | The role required to be able to authenticate. |
```

Comment:
Is there a recommended default role for full access we could recommend here and perhaps use that as a default in our config?